Header header
uint64 turn_rate
uint64 byte_rate
float64 timestamp

